### DISCUSSION TITLE
Usando Prisma em produção no Supabase

### DISCUSSION THREAD

**Discussion by gabrielsantos7** [prisma/prisma#25278] [Q&A]
### Question

Olá, pessoal, alguém poderia me ajudar? Estou tentando realizar o deploy de uma aplicação back-end que foi feita com o Prisma em um banco de dados Postgres hospedado na Supabase. Porém, ao executar o comando `npx prisma migrate deploy` no terminal, 
aparece que ele está carregando e não consegue executar as migrações para a produção, alguém poderia me ajudar?

![problem](https://github.com/user-attachments/assets/f761faaa-b94e-4044-a89e-955a1bbc218f)

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @gabrielsantos7 👋 

What does your connection url look like? Do you get any error messages outputted that you could share? If the command hangs without any error message, enabling debugging to get more insights might be helpful. You can do this by setting the DEBUG environment variable before running the command:
```
export DEBUG="*"
npx prisma migrate deploy
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
